package.json

```diff
@@ -42,8 +42,10 @@
     "react-dom": "15.6.1",
     "react-helmet": "5.1.3",
     "react-redux": "5.0.5",
-    "redux": "3.7.2",
+    "redux": "3.7.1",
+    "redux-form": "^7.0.0",
     "remote-redux-devtools": "0.5.12",
+    "shortid": "^2.2.8",
```

Comment:
Sorry, could you enlighten me on how to prevent eslint errors when using index as a key? I was initially using this:

```
{errors &&
      errors.map(err, i =>
        <span key={i}>
          {err.message}
        </span>
      )}
```